frontend/src/v5/store/store.types.ts

```diff
@@ -0,0 +1,27 @@
+/**
```

Comment:
I know that the idea for this is to have a mechanism for typing actions and not redefine them, but I think it hurts a bit  the readability of the code.